services/w3c/w3c-validation.tester.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('W3C Validation page conforms to standards with no preset and parser')
+  .get(
+    '/default.json?targetUrl=https://hsivonen.com/test/moz/messages-types/no-message.html&preset='
```

Comment:
Let's drop the empty query param from the end
```suggestion
    '/default.json?targetUrl=https://hsivonen.com/test/moz/messages-types/no-message.html'
```

Reply:
Done. And sorry to take so much of your time in reviewing this

Reply:
No worries!